common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -161,6 +161,16 @@ trait ABTestSwitches {
 
   Switch(
     ABTests,
+    "ab-acquisitions-epic-always-ask-election",
+    "This will guarantee that the epic is always displayed on election stories",
+    owners = Seq(Owner.withGithub("jranks123")),
+    safeState = On,
```

Comment:
Fair enough! I don't know which way is optimal. 😁 

I always ask just in case, since sometimes people toggle switch defaults to `on` for local testing, and unintentionally commit it. If it is intentional, all good 🤓